budcamp/Homework2/Homework2/Point.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Homework2
+{
+    struct Point
+    {
+        public double X { get; set; }
+        public double Y { get; set; }
+
+        public Point()
+        {
+            X = 0;
+            Y = 0;
+        }
+
+        public Point(int x, int y)
+        {
+            this.X = x;
+            this.Y = y;
+        }
+
+        public bool TryParsePoint(string input, out Point point)
+        {
+            point = default(Point);
+            int num1;
+            int num2;
+            string[] splitedString = new string[2];
+
+            try
+            {
+                splitedString = input.Split(',');
+                splitedString[0].Trim();
+                splitedString[1].Trim();
+
+                if (int.TryParse(splitedString[0], out num1) && int.TryParse(splitedString[1], out num2))
```

Comment:
Same thing with the declaration of `num1` and `num2`. You can do it here:  `int.TryParse(splitedString[0], out int num1`

Reply:
srry i can't declarate int num1 and int num2 inside the if block cause num1 and num2 are returning types from tryparse in the condition of if block

Reply:
have you tried to do so?

Reply:
yeap

Reply:
its then underscore and write a problem: "try to use unknown element" something like that

Reply:
![image](https://user-images.githubusercontent.com/106592587/213443946-f4ff17b8-cc33-431d-b77a-8f860d729c88.png)

Reply:
then you're missing smth, send the screen with an error to the chat.